test/docker/networktest/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # escape=`
 FROM microsoft/windowsservercore
 
-SHELL ["powershell.exe","-command"]
+SHELL ["pwsh.exe","-command"]
```

Comment:
Please don't break docker... there are tests in `docker/tests`. All tests can be can be run from windows, just run them once with the docker daemon set to windows mode and once set to Linux mode.

Reply:
This is before PSCore has been installed.  So this won't work.

Reply:
Sorry, just notice this is a test file, not a production docker file.